### ISSUE TITLE
Keeping only patients with observation period end date after 2001?

### ISSUE THREAD

**Issue by edward-burn** [oxford-pharmacoepi/cdm-etl-gold#22]
A sizeable minority of patients have observation end dates that mean they will not be included in almost any of our studies. Excluding these patients from the omop version would mean a smaller database and quicker queries.

Based on the 100k sample from cdm_gold_202301
- 7% of patients have an observation period end date before 1st January 2000
- 17% of patients have an observation period end date before 1st January 2005
- 32% of patients have an observation period end date before 1st January 2010

I don´t know what you think about this @daniprietoalhambra @ajoedicke @catalamarti? Are there any studies where you think we would need people who had left the database by 2005 for example?

**Comment by daniprietoalhambra** [comment]
I would be very happy to exclude those if this improves performance. It would be very rare for us to study patients who left GOLD before 2005


On 13 Feb 2024, at 14:54, edward-burn ***@***.***> wrote:

﻿

A sizeable minority of patients have observation end dates that mean they will not be included in almost any of our studies. Excluding these patients from the omop version would mean a smaller database and quicker queries.

Based on the 100k sample from cdm_gold_202301

  *   7% of patients have an observation period end date before 1st January 2000
  *   17% of patients have an observation period end date before 1st January 2005
  *   32% of patients have an observation period end date before 1st January 2010

I don´t know what you think about this @daniprietoalhambra<https://github.com/daniprietoalhambra> @ajoedicke<https://github.com/ajoedicke> @catalamarti<https://github.com/catalamarti>? Are there any studies where you think we would need people who had left the database by 2005 for example?

**Comment by catalamarti** [comment]
Happy with that for me!

**Comment by ajoedicke** [comment]
I agree, happy to not have them included
________________________________

**Comment by adelmestri** [comment]
Dear @daniprietoalhambra, @edward-burn, @catalamarti, @ajoedicke, sorry to be always the bad guy
We have onboarded CDM GOLD with a certain amont of patients and followup time.
Moreover, we have prepared the CDM GOLD metadata for the Erasmus Medical centre: these metadata have been published in the HMA and EMA public electronic catalogues of real-world data sources. 
All this work is based on the patients currently present in our CDM GOLD and changing that would require a big effort.
Thanks
